catalog/MDCCatalog/MDCThemePickerViewController.swift

```diff
@@ -108,6 +109,10 @@ class MDCThemePickerViewController: UIViewController, UICollectionViewDataSource
     palettesCollectionView.backgroundColor = .white
     palettesCollectionView.contentInset = UIEdgeInsets(top: 0, left: 8, bottom: 0, right: 0)
     view.addSubview(palettesCollectionView)
+    let rowWidth = view.bounds.width - cellSpacing * 2
```

Comment:
super-(super!)-nit: Ideally we would use CGRectGetWidth(view.frame) to avoid non-standardized weirdness.

Reply:
Thankfully in Swift, I believe this has been fixed.

Reply:
Indeed it has!  Disregard,.... and celebrate Swift.

Reply:
Here's the API refs.

https://developer.apple.com/documentation/coregraphics/cgrect/1454758-width